intellij-15-java-google-style.xml

```diff
@@ -0,0 +1,279 @@
+      <package name="org" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="penn2dg" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="pennconverter" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="pl" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="prefuse" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="proguard" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="repackage" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="scm" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="se" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="serp" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="simple" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="soot" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="sqlj" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="src" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="ssa" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="sun" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="sunlabs" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="tcl" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="testdata" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="testshell" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="testsuite" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="twitter4j" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="uk" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="ViolinStrings" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="weka" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="wet" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="winstone" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="woolfel" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="wowza" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="java" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="javax" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="" withSubpackages="true" static="false" />
+      <emptyLine />
+      <package name="" withSubpackages="true" static="true" />
```

Comment:
This is a great style, but static imports should be first. I know the current style has them last, but it is incorrect according to the documentation:

See this issue: 
https://github.com/google/styleguide/issues/26